docs/networks/node-ops/node-operation/node-bootstrap.md

```diff
@@ -25,15 +25,15 @@ of the `Epoch Setup Phase` to participate in the [Epoch Preparation Protocol](..
 
 ## Step 1 - Run Genesis Bootstrap
 
-<Callout type="info">
+:::info
   You will need to run this process for each node that you are operating
-</Callout>
+:::
```

Comment:
I can't do a suggestion because of the deleted line, but missing the blank lines in this admonition

Reply:
updated links to reference style